compiler/bindings/python/iree/compiler/__init__.py

```diff
@@ -7,3 +7,4 @@
 # Re-export some legacy APIs from the tools package to this top-level.
 # TODO: Deprecate and remove these names once clients are migrated.
 from .tools import *
+version = 1
```

Comment:
What is the reason we need to put this?

Reply:
Oh I think this is is a bad snaphot, not intentional